pkg/client/k8s/v1alpha1/unstructured.go

```diff
@@ -165,6 +165,15 @@ func IsNamespaceScoped(given *unstructured.Unstructured) bool {
 	return kind(given.GetKind()).isNamespaced()
 }
 
+// IsCASTemplate flags if the given unstructured instance is a CAS Template
+// scoped
```

Comment:
I think the comment should be `...CAS Template Kind` if we are trying to convey this.